From a3b4c9b9735dee8b407b7cdacc27301bc9af283b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Fri, 25 Mar 2022 10:50:03 +0100
Subject: [PATCH] do not install tests as a top-level package

Explicitly exclude tests from being found by find_packages().
Otherwise, they are installed as top-level site-packages/tests.
---
 setup.cfg | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/setup.cfg b/setup.cfg
index 9dc7a64..9dc9f11 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -39,5 +39,9 @@ install_requires =
 [options.extras_require]
 test = build; coverage; pre-commit; pytest; pytest-cov; pytest-mock
 
+[options.packages.find]
+exclude =
+    tests
+
 [bdist_wheel]
 universal=1
